rosie/chamber_of_deputies/adapter.py

```diff
@@ -15,12 +23,34 @@ def __init__(self, path):
     @property
     def dataset(self):
         self.update_datasets()
-        reimbursements = self.get_reimbursements()
+        self.get_reimbursements()
         companies = self.get_companies()
-        return pd.merge(reimbursements, companies,
-                        how='left',
-                        left_on='cnpj_cpf',
-                        right_on='cnpj')
+        self._dataset = self._dataset.merge(companies,
+                                            how='left',
+                                            left_on='cnpj_cpf',
+                                            right_on='cnpj')
+        self.prepare_dataset()
+        return self._dataset
+
+    def prepare_dataset(self):
+        self.rename_columns()
+        self.rename_categories()
+
+    def rename_columns(self):
+        columns = {v: k for k, v in COLUMNS.items()}
+        self._dataset.rename(columns=columns, inplace=True)
+
+    def rename_categories(self):
+        self._dataset['document_type'].replace({3: None}, inplace=True)
+        self._dataset['document_type'] = self._dataset['document_type'].astype(
+            'category')
+        types = ['bill_of_sale', 'simple_receipt', 'expense_made_abroad']
+        self._dataset['document_type'].cat.rename_categories(
+            types, inplace=True)
+        self._dataset['category'] = self._dataset['category'].replace(
+            {'Congressperson meal': 'Meal'})
```

Comment:
Basically we are associating the `Congressperson meal` to `Meal` to turn it in a generic way, so it will be easier to do it to other datasets.